app/lib/fact_check_config.rb

```diff
@@ -17,18 +17,30 @@ def address
     @reply_to_address
   end
 
-  def valid_subject?(subject)
-    subject.scan(@subject_pattern).length == 1
+  def contains_id?(string)
+    string.scan(@subject_pattern).length == 1
   end
 
-  def item_id_from_subject(subject)
-    match = subject.scan(@subject_pattern)
+  def item_id_from_string(string)
+    match = string.scan(@subject_pattern)
     if match.length == 1
       match[0][0]
     elsif match.length > 1
-      raise ArgumentError, "'#{subject}' has too many matches"
+      raise ArgumentError, "'#{string}' has too many matches"
     else
-      raise ArgumentError, "'#{subject}' is not a valid fact check subject"
+      raise ArgumentError, "'#{string}' does not contain any fact check ID"
     end
   end
+
+  def item_id_from_subject_or_body(subject, body = "")
+    if contains_id?(subject)
+      id = item_id_from_string(subject)
+    elsif contains_id?(body)
+      id = item_id_from_string(body)
+    else
+      raise ArgumentError, "Message does not contain any fact check ID"
+    end
```

Comment:
It feels like `item_id_from_string` could be refactored here to return `nil`, and then we could do something like:

```suggestion
    return id if (id = item_id_from_string(subject) || item_id_from_string(body))
    raise ArgumentError, "Message does not contain any fact check ID"
```

Mostly it would avoid having to `.scan` the strings twice.